layers.x86.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
For simplicity and avoiding unexpected breaks (e.g. https://runtimes.bref.sh) could we keep this file as `layers.json`? (defaults will be x86 anyway)